pom.xml

```diff
@@ -58,7 +58,7 @@ THE SOFTWARE.
         <changelist>-SNAPSHOT</changelist>
         <powermock.version>1.6.3</powermock.version>
         <jenkins.version>1.651.3</jenkins.version>
-        <java.level>7</java.level>
+        <java.level>8</java.level>
```

Comment:
If you require Java 8 you should bump the Jenkins baseline to 2.60.3, because the 2.60 LTS line is the first version that requires Java 8.

Reply:
Beat @oleg-nenashev by one second 😄

Reply:
I was reviewing from phone, so sorry for being too slow :D